index.bs

```diff
@@ -146,7 +146,6 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
         text: NewModuleEnvironment; url: sec-newmoduleenvironment
         text: NewPromiseCapability; url: sec-newpromisecapability
         text: NormalCompletion; url: sec-normalcompletion
-        text: OrdinaryCreateFromConstructor; sec: sec-ordinarycreatefromconstructor
```

Comment:
This was added in <https://github.com/whatwg/webidl/pull/987>, but isn’t used anywhere.